test/integration/ferc_dbf_extract_test.py

```diff
@@ -0,0 +1,62 @@
+"""PyTest based testing of the FERC DBF Extraction logic."""
+
+import logging
+
+import sqlalchemy as sa
+
+from pudl.extract.dbf import FercDbfReader
+
+logger = logging.getLogger(__name__)
+
+
+def test_ferc1_dbf2sqlite(ferc1_engine_dbf):
+    """Attempt to access the DBF based FERC 1 SQLite DB fixture."""
+    assert isinstance(ferc1_engine_dbf, sa.engine.Engine)  # nosec: B101
+    assert (  # nosec: B101
+        "f1_respondent_id" in sa.inspect(ferc1_engine_dbf).get_table_names()
+    )
+
+
+def test_ferc1_schema(ferc_to_sqlite_settings, pudl_datastore_fixture):
```

Comment:
Can we / should we create similar integration tests for the extraction of the other forms?  Could they be easily parameterized to just do the same thing for each of them?

Reply:
I have parametrized this so the same test is run for both `ferc1` and `ferc2`.